docs/UserGuide.md

```diff
@@ -26,26 +26,36 @@ Choose a topic from the table of contents to get started on your Wishful Shrinki
 
 1. Ensure your computer has Java `11` or above installed.
 
-2. Download the latest `wishfulShrinking.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-2/tp).
+2. Download the latest `wishfulShrinking.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-2/tp/releases).
 
 3. Copy the file to an empty folder you want to use as the _home folder_.
 
 4. **Double-click** the jar file to start the app OR start the app using **CLI** and type `java -jar <jar file name>.jar`.<br>
-   The app should look similar to the one shows below: <br>
-   ![Ui](images/Ui.png)
-
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+   The app should look similar to the one shown below:<br><br>
+   <img src="images/Ui.png" width="550" height="300">
+   <br>The image above is what Wishful Shrinking looks like after running it for the first time.<br><br><br>
+
+5. Below is a brief explanation of Wishful Shrinking's components.<br><br>
+   <img src="images/UiExplained.png" width="550" height="300">
+   <br>The image above is a labeled diagram of each of Wishful Shrinking's components. Below is a brief explanation on each of the components.<br><br>
+   
+   * List of recipes/ingredients/consumption: The left window will display either a list of recipes, a list of ingredients or the consumption list depending on your input. In the image above, it is displaying the recipe list.
+   * Command Result: The Command Result box will show the result of your input into the Command Box.
+   * Command Box: Here is where you will type all your commands.
+
+6. Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
```

Comment:
can remove the eg help since the examples are all already below?

Reply:
I think this eg is ok since it specifies that the user has to press enter also? below it doesnt specify the enter